mu-trees/addon/resolvers/glimmer-wrapper/index.js

```diff
@@ -24,21 +24,39 @@ const Resolver = DefaultResolver.extend({
 
   normalize: null,
 
-  resolve(lookupString) {
+  resolve(lookupString, referrer) {
     /*
-     * Ember paritals are looked up as templates. Here we replace the template
+     * Ember partials are looked up as templates. Here we replace the template
      * resolution with a partial resolute when appropriate. Try to keep this
      * code as "pay-go" as possible.
      */
 
     if (lookupString.indexOf('template:') === 0) {
       lookupString = this._templateToPartial(lookupString);
     }
-    return this._resolve(lookupString);
+
+    /*
+     * Ember components require their lookupString to be massaged. Make this
+     * as "pay-go" as possible.
+     */
+    if (lookupString.indexOf('template:components/') === 0) {
+      if (referrer) {
+        // make absolute
+        let appName = this.config.app.rootName || 'app';
```

Comment:
Mirrors the logic at https://github.com/glimmerjs/glimmer-resolver/blob/e33a65bc220c8ad1e31b83000e9449e8906f66ef/src/resolver.ts#L74